test_suite/cloud/test_aws.py

```diff
@@ -4,13 +4,19 @@
 import pytest
 from packaging import version
 
-from lib import test_lib
+from lib import test_lib, aws_lib
 
 
 @pytest.fixture
 def instance_data_aws_web(host):
-    instance_document_url = 'http://169.254.169.254/latest/dynamic/instance-identity/document'
-    return json.loads(host.check_output(f'curl -s {instance_document_url}'))
+    return aws_lib.get_aws_instance_identity_from_web(host)
+
+
+@pytest.fixture
+def rhel_aws_marketplace_only(host):
```

Comment:
Maybe this marker is redundant, since we already have one in the conftest.py file, that is actually more generic?